packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -242,6 +242,14 @@ export function createRococo (t: TFunction): EndpointOption {
           'Konomi Network': 'wss://rococo.konomi.tech'
         }
       },
+      {
+        info: 'rococoKylin',
+        paraId: 2004,
```

Comment:
```suggestion
        paraId: 2000,
```

As registered by you.

Reply:
Thats the thing, I technically still havent registered because I am still unclear on where to run the `setKeys` function, my own validator thats still syncing or any public validator on the rococo network?

I would assume once its syncs completely it should be the same